oak_functions/loader/src/main.rs

```diff
@@ -0,0 +1,57 @@
+//
+// Copyright 2021 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+use structopt::StructOpt;
+
+mod server;
+
+use crate::server::WasmServer;
+
+#[cfg(test)]
+mod tests;
+
+/// Command line options for the Oak loader.
+#[derive(StructOpt, Clone, Debug)]
+#[structopt(about = "Oak Functions Loader")]
+pub struct Opt {
+    #[structopt(long, default_value = "8080", help = "Server Port number.")]
+    server_port: u16,
```

Comment:
Should this be `http_listen_address` or similar? I thought we use something like that elsewhere, also it seems more explicit.